.sjs files are no longer resolved?

It's been a little while since I've used sweet.js. It doesn't seem to resolve modules to files with the `.sjs` extension anymore:

```
$ npm install es6-macros
$ ls node_modules/es6-macros/index.sjs 
node_modules/es6-macros/index.sjs

$  sjs -m es6-macros app.sjs           

/Users/james/tmp/ppp/node_modules/sweet.js/node_modules/resolve/lib/sync.js:32
    throw new Error("Cannot find module '" + x + "' from '" + y + "'");
          ^
Error: Cannot find module 'es6-macros' from '/Users/james/tmp/ppp'
```

I also tried it with just a local file with the `.sjs` extension. Are we discouraged from using that now?

--- a/src/sweet.js
+++ b/src/sweet.js
@@ -183,7 +183,10 @@
         if (moduleName[0] === '.') {
             moduleName = path.resolve(root, moduleName)
         }
-        var filename = resolveSync(moduleName, {basedir: root});
+        var filename = resolveSync(moduleName, {
+            basedir: root,
+            extensions: ['.js', '.sjs']
+        });
         return expandModule(fs.readFileSync(filename, "utf8"), undefined, {
             filename: moduleName,
             requireModule: options.requireModule || requireModule

